### DISCUSSION TITLE
Mention JSpecify in CONTRIBUTING.md

### DISCUSSION THREAD

**Discussion by Doc94** [PaperMC/Paper#11459] [Ideas]
### Is your feature request related to a problem?

N/A

### Describe the solution you'd like.

Add in the CONTRIBUTING.md a section for mention about how PR related to Paper classes need the use of jspecify rather than jetbrains-annotation

### Describe alternatives you've considered.

N/A

**Comment by kennytv** [comment]
Added in https://github.com/PaperMC/Paper/commit/fb768404aa8bdd35b95c88132b95f5cb529cc098
